docs/language_guide.md

```diff
@@ -0,0 +1,88 @@
+# The Bosatsu Programming Language
+
+## Design Philosophy and Goals
+Bosatsu has several main themes:
+
+1. correctness
+1. simplicity
+1. usability
+
+These are roughly in priority order: correctness is more important than simplicity: we don't want to sacrifice correctness for usability. Similarly, generally, we prefer simplicity to usability. Some features which can increase usability make a language considerably more complex. In general, we will be skeptical of such trades.
+
+The main user experience of the language is inspired by Python. It is a bit of a mystery that Python is currently one of the fastest growing languages as well as already being incredibly popular that it has not been more influential in language design. No other language in the top 20 most commonly used languages is very syntactically similar to Python. Put succinctly, we want Bosatsu to by a minimalistic statically typed Python.
```

Comment:
This sentence feels like it's missing the word "given" or something along those lines.